funcx_sdk/tests/utils.py

```diff
@@ -0,0 +1,76 @@
+import itertools
+import sys
+import time
+import types
+import typing as t
+
+
+def create_traceback(start: int = 0) -> types.TracebackType:
+    """
+    Dynamically create a traceback.
+
+    Builds a traceback from the top of the stack (the currently executing frame) on
+    down to the root frame.  Optionally, use start to build from an earlier stack
+    frame.
+    """
+    tb = None
+    for depth in itertools.count(start + 1, 1):
+        try:
+            frame = sys._getframe(depth)
+            tb = types.TracebackType(tb, frame, frame.f_lasti, frame.f_lineno)
+        except ValueError:
+            break
+    return tb
+
+
+def try_assert(
```

Comment:
This is copied from result-processor/tests/utils.py yes?  I suppose it's a PITA to put this into a common module in fx-common, but perhaps a TODO of getting rid of the duplication on next fx-common update?

Reply:
That's a good idea ... perhaps for a follow-up PR.